src/crimson/os/seastore/random_block_manager/rbm_device.h

```diff
@@ -152,25 +153,48 @@ class RBMDevice : public Device {
     uint16_t stream = 0) = 0;
 
   bool is_data_protection_enabled() const { return false; }
-};
 
+  mkfs_ret mkfs(device_config_t) final;
+
+  write_ertr::future<> write_rbm_header();
+
+  read_ertr::future<rbm_metadata_header_t> read_rbm_header(rbm_abs_addr addr);
+
+  using stat_device_ret =
+    read_ertr::future<seastar::stat_data>;
+  virtual stat_device_ret stat_device() = 0;
+
+  uint64_t get_journal_size() const {
+    return super.journal_size;
+  }
 
-class TestMemory : public RBMDevice {
+  void set_journal_size(uint64_t size) {
```

Comment:
IIUC `RBMDevice::set_journal_size()` is for test, group test interfaces together with `set_device_id()` and move to the very end of the class.

Reply:
Done.